elementary/monitor/dbt_project/macros/alerts/get_test_alerts.sql

```diff
@@ -68,48 +67,44 @@
         where suppression_status = 'pending'
     {% endset %}
 
-    {% set alerts_agate = run_query(select_pending_alerts_query) %}
+    {% set alerts_agate = elementary.run_query(select_pending_alerts_query) %}
+    {% set test_result_rows_agate = elementary_internal.get_result_rows_agate(days_back) %}
```

Comment:
Tiny concern - there's a chance of a race here between the two queries.
I don't think it's a big deal (worst case the samples will be missing in this case), but just worth double checking it won't cause us to crash.

Reply:
I think that given that the `select_pending_alerts_query` is built from the `alerts` model which is incremented within `edr`, and that `test_result_rows` is built within the dbt package, it should necessarily contain the result rows of what's returned from `alerts`.

What do you think?

Reply:
Makes sense to me